test/integration/cluster-init/expected/ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -46,6 +47,8 @@ periodics:
     repo: release
   interval: 12h
   labels:
+    ci-operator.openshift.io/cluster-init-controlled: "true"
+    ci-operator.openshift.io/cluster-init-controlled-newCluster: "true"
```

Comment:
nit: labels are a mapping. Why don't we do something like

```yaml
  labels:
    ci-operator.openshift.io/cluster-init-controlled: "true"
    ci-operator.openshift.io/cluster-init: "newCluster"
```

We can keep the `ci-operator.openshift.io/cluster-init-controlled` as the one that switches around between `newly-generated` etc and the other can just be an identifying label for selection.